src/format.ts

```diff
@@ -19,14 +19,29 @@ export interface PluginOptions {
    * omit formatted files
    */
   filter?: boolean;
+  /**
+   * include rules from Prettier config files, e.g. .prettierrc
+   */
+  configFile?: boolean;
 }
 
 export function format(
   prettier_options: Options = {},
-  { reporter = Reporter.Warning, filter = false }: PluginOptions = {},
+  {
+    reporter = Reporter.Warning,
+    filter = false,
+    configFile: config_file = true,
+  }: PluginOptions = {},
 ) {
-  return create_transform((text, filename) => {
+  return create_transform(async (text, filename) => {
+    // tslint:disable-next-line:strict-type-predicates
+    const resolved_config =
+      config_file && typeof prettier.resolveConfig === 'function'
```

Comment:
`// tslint:disable-next-line:strict-type-predicates` should be here. 😆

Reply:
Prettier moved it around on me after I linted 😀